token/js/src/extensions/transferFee.ts

```diff
@@ -0,0 +1,24 @@
+import { Layout, struct } from '@solana/buffer-layout';
+import { Mint } from '../state/mint';
+import { ExtensionType, getExtensionData } from './extensionType';
+
+
+/** TransferFeeAmount as stored by the program */
+export interface TransferFeeAmount {
+    feeAmount: Number;
```

Comment:
Since the fee is given by a `u64` in the code, we should use a `bigint` here, similar to the `amount` in `Account`